### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by toddmedema**
- lock to portrait mode
- fullscreen: true
- iOS specific fullscreen params + set export compliance no-encryption flag

### CODE REVIEW

**Comment by smartin015** [comment]
Please fix the merge conflicts I made so I can review :P

**Comment by toddmedema** [comment]
@smartin015 updated

File: config.xml [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
+    <allow-intent href="market:*" />
+  </platform>
+  <platform name="ios">
+    <allow-intent href="itms:*" />
+    <allow-intent href="itms-apps:*" />
+  </platform>
+
+  <preference name="Fullscreen" value="true" />
+  <preference name="BackgroundColor" value="0x00000000" />
+  <preference name="Orientation" value="Portrait" />
```
Comment (smartin015) [inline config.xml]: You spelled "Portait" wrong :P

**Comment by toddmedema** [comment]
@smartin015 ready for review round 2, fixes several iOS bugs

**Comment by toddmedema** [comment]
Closing, a subset of https://github.com/Fabricate-IO/expedition-app/pull/15

**Pull request closed without merging**